cursor.toml

```diff
@@ -0,0 +1,2 @@
+txDigest = "9hH5mDKNegxazEDAXzACauQdiG4KZqR1VTQoBhgVpQzB"
```

Comment:
Do we really want to commit this file ? I presume we want to remove it altogether from the repository, to not conflict with previous state history.